Firebase/android/app/src/main/res/values/login_styles.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <style name="Login"/>
+
+  <style name="Login.Layout">
+    <item name="android:background">@color/colorPrimary</item>
+    <item name="android:gravity">top</item>
+    <item name="android:orientation">vertical</item>
+  </style>
+
+  <style name="Login.Disclaimer">
+    <item name="android:layout_marginBottom">@dimen/login_disclaimer_bottom_margin</item>
+    <item name="android:layout_marginLeft">@dimen/login_disclaimer_horizontal_margin</item>
+    <item name="android:layout_marginRight">@dimen/login_disclaimer_horizontal_margin</item>
+    <item name="android:layout_marginTop">@dimen/login_disclaimer_top_margin</item>
+    <item name="android:gravity">center</item>
+    <item name="android:textColor">@color/white</item>
+    <item name="android:textSize">14sp</item>
+  </style>
+
+  <style name="Login.AppIcon">
+    <item name="android:layout_gravity">center</item>
+    <item name="android:minHeight">@dimen/login_app_icon_size</item>
+    <item name="android:minWidth">@dimen/login_app_icon_size</item>
```

Comment:
I think minW/H should also go in the layout, but not strongly one way or the other